daemon/plugin_p.h

```diff
@@ -66,7 +66,7 @@ class PluginHelper : public QPluginLoader
             return 0;
         }
         if (plugin->name().endsWith(QStringLiteral(".test"), Qt::CaseInsensitive) != m_autotestMode) {
-            qCDebug(category) << "ignoring plugin:" << fileName() << "due to mode";
+            qCDebug(category) << "ignoring plugin:" << fileName() << "because we are in testing mode and it isn't a test plugin";
```

Comment:
The message is not correct, since we may be in non-test mode but the plugin is a test plugin, which would still cause this codepath to be hit.